recommendation/corewcf.channels.json

```diff
@@ -0,0 +1,49 @@
+{
+  "Name": "System.ServiceModel.Channels",
```

Comment:
I believe the file name should match this line here. Otherwise this may lead to problems when associating rules files with namespaces.